benchexec/tools/abc.py

```diff
@@ -24,7 +25,33 @@ def name(self):
         return "ABC"
 
     def cmdline(self, executable, options, task, rlimits):
-        return [executable] + options + [task.single_input_file]
+        return (
+            [executable]
+            + ["-c", "&r {}; &put".format(task.single_input_file)]
```

Comment:
This is a somewhat unexpected way to call a tool, maybe a comment would be nice that explains why this is done?

And is this backwards compatible or does it change the behavior of the tool compared to the previous version?

Using f-strings would be nicer than `.format()`.

Reply:
> This is a somewhat unexpected way to call a tool, maybe a comment would be nice that explains why this is done?

I've added some documentation.

> And is this backwards compatible or does it change the behavior of the tool compared to the previous version?

Yes, this is backward compatible.

> Using f-strings would be nicer than `.format()`.

Done in c2ccbdd.